website/blog/2018-12-21-state-of-react-native-community.md

```diff
@@ -0,0 +1,51 @@
+---
+title: The State the React Native Community in 2018
+author: Lorenzo Sciandra
+authorTitle: Core Maintainer & React Native Developer
+authorURL: https://github.com/kelset
+authorImageURL: https://avatars2.githubusercontent.com/u/16104054?s=460&v=4
+authorTwitter: kelset
+category: announcements
+---
+
+This year we made a large number of changes about how we develop and communicate about React Native; we hope that in two years from now we can look back and see that this shift was a turning point for React Native.
```

Comment:
```suggestion
In 2018 the React Native Community made a number of changes about how we develop and communicate about React Native. We believe that a few years from now we will look back and see that this shift was a turning point for React Native.
```

- Specify the year
- Remove "large" (how do you quantify this)
- Remove semicolon
- Change hope to belief
- Change two years to a few years